src/site/filters/liquid.js

```diff
@@ -206,6 +206,16 @@ module.exports = function registerFilters() {
     return _.slice(arr, startIndex);
   };
 
+  liquid.filters.formatSharableLinkID = (id, description) => {
+    if (!description) return id;
+    return `${description
+      .replace(/[^a-zA-Z ]/g, '')
+      .substring(0, 30)
+      .toLowerCase()
+      .split(' ')
+      .join('-')}-${id}`;
+  };
```

Comment:
a couple suggestions on this 

- this is golden for a unit test because it has such a straightforward input/output. can we add a few for this at `src/site/filters/liquid.unit.spec.js`?
- there are a lot of manipulations to `description` inside of the template string so it's kind of hard to read. maybe try something like

```js
const hyphenatedDesc = description
      .replace(/[^a-zA-Z ]/g, '')
      .substring(0, 30)
      .toLowerCase()
      .split(' ')
      .join('-');

// you might be able to get the same result through _.kebabCase(description) 

// you can also call other liquid filters here, like liquid.filters.escape()

return `${hyphenatedDesc}-${id}`;
```

Reply:
done